src/ibusengine.c

```diff
@@ -391,6 +394,21 @@ ibus_engine_class_init (IBusEngineClass *class)
                         G_PARAM_READWRITE |
                         G_PARAM_CONSTRUCT_ONLY));
 
+    /**
+     * IBusEngine:needs-surrounding-text:
+     *
+     * Whether the IBusEngine needs to receive updates when surrounding text
+     * changes.
```

Comment:
Since the comment only will be in the API document, I think the comment still does not help users.
We need to explain that "RequireSurroundingText" D-Bus signal will be called in ibus-daemon with focus-in/out events without calling ibus_engine_get_surrounding_text() when ths property is enabled and the this property can be configured in the engine constructor only.